libs/models/src/lib/state/state-model.ts

```diff
@@ -0,0 +1,19 @@
+import { AppErrorLogs } from './log-message';
+import { AppLeaderboards } from './leaderboard';
+import { AppPlayerLogs, AppPlayers } from './player';
+import { AppRaces } from './race';
+import { AppTournaments } from './tournament';
+
+export enum AppIdNumberType {
```

Comment:
Why do we have `AppIdNumberType` defined within `state-model`?
Can we move it to its own file?

Reply:
Moved aeb12cc93d2ac598fa64ebcccfc3c07d30251b2b